src/modules/olMap/handler/routing/OlRoutingHandler.js

```diff
@@ -0,0 +1,652 @@
+		this._modifyInteraction = null;
+		this._translateInteraction = null;
+		this._selectInteraction = null;
+		// other
+		this._catId = null;
+		this._currentRoutingResponse = null;
+		this._promiseQueue = new PromiseQueue();
+		this._registeredObservers = [];
+		this._mapListeners = [];
+		this._helpTooltip = new HelpTooltip();
+		this._helpTooltip.messageProvideFunction = messageProvide;
+	}
+
+	/**
+	 * Activates the Handler.
+	 * @override
+	 */
+	onActivate(olMap) {
+		this._map = olMap;
+
+		this._alternativeRouteLayer = this._createLayer(RoutingLayerIds.ROUTE_ALTERNATIVE);
+		this._routeLayer = this._createLayer(RoutingLayerIds.ROUTE);
+		this._routeLayerCopy = this._createLayer(RoutingLayerIds.ROUTE_COPY);
+		this._highlightLayer = this._createLayer(RoutingLayerIds.HIGHLIGHT);
+		this._interactionLayer = this._createLayer(RoutingLayerIds.INTERACTION);
+
+		this._routingLayerGroup = new LayerGroup({
+			layers: [this._alternativeRouteLayer, this._routeLayer, this._routeLayerCopy, this._highlightLayer, this._interactionLayer]
+		});
+
+		this._translateInteraction = this._createTranslate(this._interactionLayer);
+		this._map.addInteraction(this._translateInteraction);
+		this._selectInteraction = this._createSelect(this._interactionLayer, this._alternativeRouteLayer);
+		this._map.addInteraction(this._selectInteraction);
+
+		if (!this._environmentService.isTouch()) {
+			this._modifyInteraction = this._createModify();
+			this._map.addInteraction(this._modifyInteraction);
+		}
+
+		this._registeredObservers = this._register(this._storeService.getStore());
+		this._mapListeners.push(
+			olMap.on(
+				MapBrowserEventType.POINTERMOVE,
+				this._newPointerMoveHandler(olMap, this._interactionLayer, this._alternativeRouteLayer, this._routeLayerCopy)
+			)
+		);
+		return this._routingLayerGroup;
+	}
+
+	_getPointerMoveGetFeaturesAtPixelOptions(interactionLayer, alternativeRouteLayer, routeLayerCopy) {
+		return {
+			layerFilter: (layer) => [interactionLayer, alternativeRouteLayer, routeLayerCopy].includes(layer),
+			hitTolerance: 5
+		};
+	}
+
+	_newPointerMoveHandler(map, interactionLayer, alternativeRouteLayer, routeLayerCopy) {
+		const updateModifyActivity = (feature) => {
+			if (feature.get(ROUTING_FEATURE_TYPE) === RoutingFeatureTypes.ROUTE_SEGMENT) {
+				this._setModifyActive(true);
+			} else {
+				this._setModifyActive(false);
+			}
+		};
+		const updateCursor = (feature) => {
+			if (feature.get(ROUTING_FEATURE_TYPE) === RoutingFeatureTypes.ROUTE_SEGMENT) {
+				map.getTarget().style.cursor = 'grab';
+			} else {
+				map.getTarget().style.cursor = 'pointer';
+			}
+		};
+		return (event) => {
+			this._helpTooltip.deactivate();
+
+			if (!event?.dragging) {
+				// $(element).popover('destroy');
+
+				const pixel = map.getEventPixel(event.originalEvent);
+				const hit = map.getFeaturesAtPixel(
+					pixel,
+					this._getPointerMoveGetFeaturesAtPixelOptions(interactionLayer, alternativeRouteLayer, routeLayerCopy)
```

Comment:
a hard readable method name...but a better suggestion is even harder :D  
How about 
```suggestion
					this._getFeaturesAtPixelOptionsForPointerMove(interactionLayer, alternativeRouteLayer, routeLayerCopy)
```